src/uct/ib/rc/base/rc_ep.c

```diff
@@ -79,12 +79,17 @@ void uct_rc_txqp_vfs_populate(uct_rc_txqp_t *txqp, void *parent_obj)
                             &txqp->available, UCS_VFS_TYPE_I16, "available");
 }
 
-ucs_status_t uct_rc_fc_init(uct_rc_fc_t *fc, int16_t winsize
+void uct_rc_fc_reset(const uct_rc_iface_t *iface, uct_rc_fc_t *fc)
+{
+    fc->fc_wnd = iface->config.fc_wnd_size;
+}
+
+ucs_status_t uct_rc_fc_init(uct_rc_fc_t *fc, const uct_rc_iface_t *iface
```

Comment:
ucs_status_t uct_rc_iface_fc_init(uct_rc_iface_t *iface, uct_rc_fc_t *fc, ..)

Reply:
shall we move it to iface file then?

Reply:
yes

Reply:
done